src/common/components/entry-list/index.tsx

```diff
@@ -101,10 +101,13 @@ export class EntryListContent extends Component<Props, State> {
   }
 
   render() {
-    const { entries, promotedEntries, global, activeUser, loading } = this.props;
+    const { entries, promotedEntries, global, activeUser, loading, location } = this.props;
     const { filter, tag } = global;
     const { mutedUsers, loadingMutedUsers } = this.state;
     let dataToRender = entries;
+    if (location.pathname.startsWith("/promoted")) {
```

Comment:
`includes` or `indexOf` are better than `startsWith` from what I remember startsWith not fully supported on some browsers.

Reply:
updated @feruzm